docs/vc-rest/README.md

```diff
@@ -0,0 +1,175 @@
+## REST API ENDPOINTS 
+
+### Create issuer profile  - POST /profile
+This is the first endpoint to continue the flow the in the edge-service. 
+Mandatory fields: name, did, uri, signatureType, creator
```

Comment:
not all the fields are obvious from the description. suggest to define (e.g., name).

Reply:
updated